### DISCUSSION TITLE
第4节课笔记(8班)

### DISCUSSION THREAD

**Discussion by Hongru0306** [InternLM/Tutorial#280] [笔记]
请在下方提交笔记链接,格式为
-->报名序号+连接:
例如:213+https://kvudif1helh.feishu.cn/docx/...

**Comment by Yujin320** [comment]
587+https://github.com/Yujin320/InternLM-hyx-Learning/blob/main/handwrite04.md

**Comment by NYYY0421** [comment]
501+https://github.com/NYYY0421/InternLM_Notes/blob/main/note4/note4.md

**Comment by zhaoxin111** [comment]
445+https://app.yinxiang.com/fx/8c8823d6-cbed-4f2f-81bd-d6d6ee47aba9

**Comment by Cyydz** [comment]
527+https://www.wolai.com/3oPwED5Ky7L7a9Cfa2vCA6

**Comment by jamiechoi1995** [comment]
554+https://blog.csdn.net/sinat_26253653/article/details/135566396

**Comment by baijiesong** [comment]
509+https://github.com/baijiesong/InternLM_Learning/blob/main/%E7%AC%AC%E5%9B%9B%E8%8A%82%E7%AC%94%E8%AE%B0.md

**Comment by L-369-maha** [comment]
521+https://github.com/L-369-maha/InternLM/blob/main/Four-Xtuner/Note.md

**Comment by superkong001** [comment]
545+https://github.com/superkong001/InternLM_Learning/blob/main/homework3_XTuner_note.md

**Comment by hzsun1995** [comment]
446+https://github.com/hzsun1995/internlm-course/blob/main/course4_note.md

**Comment by Alias-z** [comment]
388+http://t.csdnimg.cn/g4XQa

**Comment by 00z768** [comment]
1095 + https://github.com/00z768/model/blob/main/xturner微调工具箱--微调大模型.md

**Comment by JieGenius** [comment]
555+https://github.com/JieGenius/InternLMLearning/blob/main/lesson4/note.md

**Comment by BJUT-WangZhengTao** [comment]
371+https://blog.csdn.net/m0_54169072/article/details/135579965

**Comment by carlinl** [comment]
596 + https://pt35a5ibej.feishu.cn/docx/Agz2dVlu8okpeExTi82c7HbznMh?from=from_copylink

**Comment by sunxiaobei** [comment]
585+https://github.com/sunxiaobei/InternLM_tutorial/blob/main/notes/Day04.md

**Comment by Doublefei-Giser** [comment]
409+https://blog.51cto.com/u_16507376/9392123

**Comment by ZPfree** [comment]
605+https://github.com/ZPfree/homework/blob/main/4note.md
